test/unit/test-reader/index.js

```diff
@@ -100,6 +100,14 @@ describe('test-reader', () => {
         });
     });
 
+    it('should run with empty sets', () => {
+        const config = {
+            getBrowserIds: _.constant([])
```

Comment:
I think the reason under that call - https://github.com/gemini-testing/gemini/blob/master/lib/test-reader/set-collection.js#L18